drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -302,34 +294,20 @@ def save_new_enrollment!
   end
 
   def save_in_progress!
-    # "WIP" Enrollments have a null ProjectID column, so that they don't
-    # get pulled into reporting. The Project database ID is stored in the hmis_wips table.
-    saved_project_id = project.id
-
+    # "WIP" Enrollments have a null ProjectID column, so that they aren't included in reporting.
     self.project_id = nil
-    self.wip = Hmis::Wip.find_or_create_by(
-      {
-        source: self,
-        project_id: saved_project_id,
-        client_id: client.id,
-        date: entry_date,
-      },
-    )
     save!(validate: false)
   end
   alias save_in_progress save_in_progress!
 
   def save_not_in_progress!
-    transaction do
-      self.project_id = project_id || project.project_id
-      wip&.destroy!
-      save!
-    end
+    self.project_id = project.project_id
+    save!
   end
   alias save_not_in_progress save_not_in_progress!
 
   def in_progress?
-    project_id.nil?
+    read_attribute(:ProjectID).nil?
```

Comment:
Whats the difference here?

Reply:
Okay I see. I think its fine I was just wondering if `read_attribute` had some impact like not reading from local changes on the record or something like that.  Fine with this, or `ProjectID.nil?` to emphasize the casing..